### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by sciencefixion**
### New Feature Submissions:
* [X] Have you linted your code locally prior to submission?

### Testing:
* [X] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed?

**Test coverage** = 100%
--------------------------------------------------------------------------------

### CODE REVIEW

**Review by janegreene — APPROVED** [review]
Looks fantastic, nice job!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## All Submissions:
```
Comment (kathytbui) [inline .github/pull_request_template.md:1]: Love the template! Didn't know we could do this

**Review by kathytbui — APPROVED** [review]
Looks good, glad we got all of our environment set up.

**Pull request merged**